### PR TITLE
Vertical tabs responsive container

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-tabs?_sp.spa=pr-vertical-tabs-responsive-container](https://developer.blackbaud.com/skyux-tabs?_sp.spa=pr-vertical-tabs-responsive-container)

File: src/app/public/modules/vertical-tabset/vertical-tab.component.ts [hunk 1]
```diff
@@ -169,7 +218,20 @@ export class SkyVerticalTabComponent implements OnInit, OnDestroy {
     }
   }
 
+  @HostListener('window:resize', ['$event'])
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/vertical-tabset/vertical-tab.component.ts:221]: Do we need the `event`? Can we just remove it?

**Review by Blackbaud-AlexKingman — COMMENTED** [review]
Looks good. Two minor comments.

File: src/app/public/modules/vertical-tabset/vertical-tab.component.html [hunk 2]
```diff
@@ -33,7 +33,10 @@
   </div>
 </a>
 
-<div #tabContentWrapper>
+<div
+  class="sky-vertical-tab-content-wrapper"
```
[the remark this answers is not in the document]
Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/vertical-tabset/vertical-tab.component.html]: That was a product of some early implementation stuff that was removed - I just missed removing this line. Removed.

File: src/app/public/modules/vertical-tabset/vertical-tab.component.ts (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/vertical-tabset/vertical-tab.component.ts]: We don't need it, good catch. Removed.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-tabs?_sp.spa=pr-vertical-tabs-responsive-container](https://developer.blackbaud.com/skyux-tabs?_sp.spa=pr-vertical-tabs-responsive-container)

**Review by Blackbaud-AlexKingman — DISMISSED** [review]
🚢  🇮🇹

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-tabs?_sp.spa=pr-vertical-tabs-responsive-container](https://developer.blackbaud.com/skyux-tabs?_sp.spa=pr-vertical-tabs-responsive-container)

**Review by Blackbaud-AlexKingman — APPROVED** [review]

**Pull request merged**